events/README.md

```diff
@@ -0,0 +1,472 @@
+    "region": "in"
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "id": "<id>",
+        "name": "<event_name>",
+        "enabled": true,
+        "description": "This is a sample description for the event",
+        "customer": "<customer_id>",
+        "recording_info": {
+          "enabled": false
+        },
+        "template_id": "<template_id>",
+        "region": "in",
+        "created_at": "YYYY-MM-DDTHH:MM:SS.sssZ",
+        "updated_at": "YYYY-MM-DDTHH:MM:SS.sssZ"
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't create event. Please try again later'`
+
+## GET /events
+
+Get all the created events.
+
+- Params:
+  - enabled: true/false
+  - hits: Number of events in one response
+  - offset: Start of response
+    - ex: For 1-10 hits: 10, offset: 0; 11-20 hits: 10, offset: 10;
+- Query
+  - None
+- Body
+  - None
+- Headers
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```jsx
+      {
+      	"limit": 10,
+      	"data": [
+      		<event_object>,
+      		<event_object>,
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't get events. Please try again later'`
+
+## POST - /events/join
+
+Generate an auth token for a peer to join a event.
+
+To join the event (for guests, host, maven, moderator)
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - Required: **`event_id=[string]`** (The ID of the event to which the user belongs)
+  - Required: **`user_id=[string]`** (The ID of the user for whom the token is being generated)
+  - Required: **`role=[string]`** (The role of the user in the event)
+  ```json
+  {
+    "event_id": "<event_id>",
+    "user_id": "<user_id>",
+    "role": "<role>"
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "token": "<auth_token>",
+        "msg": "Token generated successfully!",
+        "success": true
+      }
+      ```
+- **Error Response:**
+  - **Code:** 500
+    - **Content**
+      **`{ "msg": "Some error occured!", "success": false }`**
```

Comment:
at some places we are calling it `message`. it will be better if we call t he field `message` here as well instead of `msg` to stay consistent.